bin/prod/update.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if ps aux| grep -v grep |grep --quiet rclone; then
+  echo "rclone is running, not redeploying at this time"
+  echo "in future, consider adding delays/retries here"
+  exit 1
+else
+  echo rclone is not running, ok to deploy
+fi
+
+
+echo updating code...
+sudo bash -c "(cd /root/motuz && git pull)" # # TODO unhardcode motuz dir
+# TODO - make sure we are in the right branch?
+echo running redeploy script...
+# TODO - disable 'sleep' in redeploy script if calling from here, to minimize downtime
+sudo bash -c "(cd /root/motuz && bin/prod/redeploy.sh)"
+echo Done.
+
+
+
```

Comment:
This file is quite specific to FredHutch. Do we want to create some separation between that and motuz? If so, I recommend starting a separate repository for this purpose. Otherwise, the file looks good for now.

Reply:
It is quite specific to FH. But it would be tricky to add it to a different repo. It is the motuz repo that we want to trigger the CI/CD. That's not how CircleCI works by default.

Regarding triggering a build on pull requests, please see https://github.com/FredHutch/motuz/blob/experiment/ci-cd/circle.yml#L17 and https://github.com/FredHutch/motuz/blob/experiment/ci-cd/circle.yml#L26-L29 ; the CI-CD is only triggered on pushes to master.